docs/build.sbt

```diff
@@ -1,6 +1,4 @@
-
-val PlayVersion = "2.6.0"
-val AkkaVersion = "2.5.3"
+import Dependencies._
```

Comment:
Yes... this is the reason this issue is hard :smile: I'm not sure how to share `Dependencies` between the two builds without duplicating it.

@erip if you can find a solution, that would be amazing. I think we can ignore the whitelist issues until then, as this is the crux of the problem, and those are minor details.